code/_onclick/hud/new_player.dm

```diff
@@ -47,6 +47,7 @@
 		return
 	flick("[base_icon_state]_pressed", src)
 	update_appearance(UPDATE_ICON)
+	SEND_SOUND(hud.mymob, sound('sound/effects/save.ogg'))
```

Comment:
This probably isn't necessary, but just because it startles me sometimes when I have my volume too set high:
```suggestion
	SEND_SOUND(hud.mymob, sound('sound/effects/save.ogg', volume = 50))
```

Reply:
Set it to 75, 50 is a bit too quiet